### ISSUE TITLE
Selected elements not announced by screen reader in ng-select pills

### ISSUE THREAD

**Issue by razaamjad** [ng-select/ng-select#2206]
**Describe the bug**
When using the ng-select component with the "multiple" attribute, selected elements are displayed as pills. However, navigating through the pills using a screen reader or tab doesn't happen also the selected element is not announced, making it difficult for users with visual impairments to understand the current selection.

**Reproducbile example**
[Example](https://stackblitz.com) which reproduces described behaviour

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Use the ng-select component with the "multiple" attribute and default selected elements.
2. Load the ng-select component and observe that the default selected elements are displayed as pills.
3. Navigate through the pills using a screen reader (e.g. NVDA, VoiceOver) or tab that doesn’t happen.
4. Notice that the default selected element is not announced by the screen reader.

**Expected behavior**
Navigating through the pills using a screen reader or tab should happen, each selected element, including the default selected element, should be announced by the screen reader.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [ALL]
 - Browser [All]
 - Version [Latest]

**Smartphone (please complete the following information):**
 - Device: [ALL]
 - OS: [ALL]
 - Browser [ALL]
 - Version [Latest]

**Additional context**
Actual behavior: Navigating through the pills using a screen reader or tab doesn't happen, the default selected element is not announced by the screen reader.

Impact: Users with visual impairments may have difficulty understanding the current selection when using the ng-select component with the "multiple" attribute and default selected elements.
